HW4_Shtompel/protein_tools.py

```diff
@@ -0,0 +1,200 @@
+"""
+Global variables:
+- AA_ALPHABET — a dictionary variable that contains a list of proteinogenic aminoacids classes.
+- ALL_AMINOACIDS — a set variable that contains a list of all proteinogenic aminoacids.
+- AMINO_ACIDS_MASSES — a dictionary variable that contains masses of all proteinogenic aminoacids.
+"""
+
+AA_ALPHABET = {'Nonpolar': ['G', 'A', 'V', 'I', 'L', 'P'],
+                'Polar uncharged': ['S', 'T', 'C', 'M', 'N', 'Q'],
+                'Aromatic': ['F', 'W', 'Y'],
+                'Polar with negative charge': ['D', 'E'],
+                'Polar with positive charge': ['K', 'R', 'H']
+                }
+
+ALL_AMINOACIDS = set(('G', 'A', 'V', 'I', 'L', 'P', 'S', 'T', 'C', 'M', 'N', 'Q', 'F', 'W', 'Y', 'D', 'E', 'K', 'R', 'H'))
+
+AMINO_ACIDS_MASSES = {
+    'G': 57.05, 'A': 71.08, 'S': 87.08, 'P': 97.12, 'V': 99.13,
+    'T': 101.1, 'C': 103.1, 'L': 113.2, 'I': 113.2, 'N': 114.1,
+    'D': 115.1, 'Q': 128.1, 'K': 128.2, 'E': 129.1, 'M': 131.2,
+    'H': 137.1, 'F': 147.2, 'R': 156.2, 'Y': 163.2, 'W': 186.2    
+}
+
+
+def is_protein(seq: str) -> bool:
+    """
+    Input: a protein sequence (a str type).
+    Output: boolean value.
+    'is_protein' function check if the sequence contains only letters in the upper case.
+    """
+    if seq.isalpha() and seq.isupper():
+        return True

```

Comment:
А что если else? Возвращает None, хотя логично что должен бы False. Тут можно было даже не мудрить:
```suggestion
        return  seq.isalpha() and seq.isupper()

```